otter/convergence/service.py

```diff
@@ -148,13 +149,30 @@ def execute_convergence(tenant_id, group_id,
         elif group_state.status == ScalingGroupStatus.ERROR:
             yield Effect(UpdateGroupStatus(scaling_group=scaling_group,
                                            status=ScalingGroupStatus.ACTIVE))
+            yield cf_msg('group-status-active',
+                         status=ScalingGroupStatus.ACTIVE.name)
     elif worst_status == StepResult.FAILURE:
         yield Effect(UpdateGroupStatus(scaling_group=scaling_group,
                                        status=ScalingGroupStatus.ERROR))
+        yield cf_msg(
+            'group-status-error', status=ScalingGroupStatus.ERROR.name,
+            reasons=','.join(worst_reasons_from_sorted_results(priority)))
 
     yield do_return(worst_status)
 
 
+def worst_reasons_from_sorted_results(results):
+    """
+    Return worst reasons from list of results sorted based on severity of
+    StepResults
+    """
+    for result, reasons in results:
+        if result == StepResult.FAILURE:
+            yield ','.join(reasons)
```

Comment:
also, those non-strings can have exception info, which probably isn't something we want to expose to users